<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>building</class>
 <widget class="QWidget" name="building">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>630</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>思源黑体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Building</string>
  </property>
  <widget class="QGroupBox" name="groupBox_eles">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>430</width>
     <height>300</height>
    </rect>
   </property>
   <property name="title">
    <string>Elevators</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_btns">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>430</width>
     <height>280</height>
    </rect>
   </property>
   <property name="title">
    <string>Floor Buttons</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_bar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>430</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>就绪</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
